src/main/java/com/google/alpollo/ConfigHelper.java

```diff
@@ -11,32 +11,40 @@
 final public class ConfigHelper {
   private static final String CONFIG_FILE_PATH = "/WEB-INF/config.json";
   private static final Gson gson = new Gson();
+  public enum SENSITIVE_DATA { PROJECT_ID, API_KEY }
 
   /**
    * Class which represents the content of configuration file,
    * used only to get Json object from file.
    */
   private static final class ConfigInfo {
     private String projectID;
+    private String apiKey;
 
     public String getProjectID() {
       return projectID;
     }
+    public String getApiKey() { return apiKey; }
   }
 
   private ConfigHelper() {
     throw new RuntimeException("Instantiation of ConfigHelper is not allowed!");
   }
 
   /**
-   * Retrieves projectID from configuration file.
-   * @return projectID; {@code null} if configuration file wasn't found
-   * or it was incorrect/didn't have projectID field
+   * Retrieves sensitive from configuration file.
+   * @return String; {@code null} if configuration file wasn't found
+   * or it was incorrect/didn't have this field
```

Comment:
Full stop at the end of this sentence. (Finally I'm not the only one who forgets)